### PR TITLE
browse view all (top level) sorting functionality

### PR DESCRIPTION
**Pull request by pgandhizaizi**
## Changes in this PR

1. added a new pytest fixture to create browse file records for testing in conftest.py, 
2. modified browse_data query to accommodate sorting orders, added a new function _build_browse_sorting_orders in queries.py, 
3. modified /browse route to put example sorting order details in routes.py, 
4. created new test cases for browse sorting in test_queries.py

## JIRA ticket

AYR-598

### After

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

File: app/main/db/queries.py [hunk 1]
```diff
@@ -151,8 +168,8 @@ def _build_transferring_body_view_query(transferring_body_id):
             func.count(func.distinct(File.FileId)).label("records_held"),
         )
         .join(Consignment, Consignment.ConsignmentId == File.ConsignmentId)
-        .join(Body, Body.BodyId == Consignment.BodyId)
         .join(Series, Series.SeriesId == Consignment.SeriesId)
+        .join(Body, Body.BodyId == Series.BodyId)
```
Comment (anthonyhashemi) [inline app/main/db/queries.py:172]: thanks for fixing these joins

File: app/tests/test_queries.py [hunk 2]
```diff
@@ -584,6 +584,462 @@ def test_browse_with_date_from_and_to_filter_no_result(
 
         assert result.items == []
 
+    def test_browse_with_transferring_body_sorting_a_to_z(
```
Comment (anthonyhashemi) [inline app/tests/test_queries.py:587]: Like we discussed before, these tests could possibly be simpler and as effective at a lower level by moving the execution of the query (using pagination) to the route itself. Then we could test all these sorting edge cases without needing to worry about pagination and requiring lots of test objects.

This can be addressed when we tackle the ticket to break out the body route into distinct routes

**Review by anthonyhashemi — APPROVED** [review]
Looks good to me 👍

File: app/main/db/queries.py (hunk 1 above)
Comment (pgandhizaizi) [reply app/main/db/queries.py:172]: you are welcome

File: app/tests/test_queries.py (hunk 2 above)
Comment (pgandhizaizi) [reply app/tests/test_queries.py:587]: yes, I agree , we should address when we create distinct routes
